lean-toolchain

```diff
@@ -1 +1 @@
-leanprover/lean4:v4.19.0-rc3
+leanprover/lean4:v4.19.0-rc3
```

Comment:
```suggestion
leanprover/lean4:v4.19.0-rc3
```

Reply:
Lol will that add the newline I wonder?

Reply:
Hmm "suggestion is invalid because it doesn't change the code" :-/

Reply:
@pitmonticone is there some bug meaning that something is removing the newline at the end of `lean-toolchain`?

Reply:
The reason a newline was not added to the `lean-toolchain` file (also in FLT#398 and FLT#413) is that the change was neither made nor saved manually. In VS Code, the newline is automatically inserted only upon saving the file. 

If there is a new toolchain version available and you run `lake update` that file is changed automatically and the `files.insertFinalNewline` isn't enforced.